rootfs/etc/lua/haproxy-lua-http.lua

```diff
@@ -28,7 +28,7 @@
 --
 -- SPDX-License-Identifier: Apache-2.0
 
-local _author = "Adis Nezirovic <anezirovic@haproxy.com>"
+local _author = "Adis Nezirović <anezirovic@haproxy.com>"
```

Comment:
This matches the copyright on line 14 above. Unless there's a specific reason it shouldn't, in which case a comment may be warranted...